src/NuGet.Services.AzureSearch/ScoringProfiles/NewlyPublishedAndDownloadCountBoosterProfile.cs

```diff
@@ -1,33 +1,42 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using Microsoft.Azure.Search.Models;
 using NuGet.Services.AzureSearch.SearchService;
 
 namespace NuGet.Services.AzureSearch.ScoringProfiles
 {
-    public class DownloadCountBoosterProfile
+    public class NewlyPublishedAndDownloadCountBoosterProfile
     {
-        public const string Name = "download_count_scoring_profile";
+        public const string Name = "newly_published_and_download_count_scoring_profile";
 
         public static readonly ScoringProfile Instance = new ScoringProfile(
             Name,
             textWeights: new TextWeights
             {
                 Weights = new Dictionary<string, double>
                 {
-                    { IndexFields.PackageId, 10 },              // Exact match of the package id should be boosted higher.
-                    { IndexFields.TokenizedPackageId, 5 }
+                    { IndexFields.PackageId, 10 },              // Exact match of the package id should be boosted the highest.
+                    { IndexFields.TokenizedPackageId, 5 },
                 }
             },
             functions: new List<ScoringFunction>()
             {
+                new FreshnessScoringFunction(
+                    fieldName: IndexFields.Published,
+                    boost: 5.0,
+                    boostingDuration: new TimeSpan(365, 0, 0, 0),
+                    interpolation: ScoringFunctionInterpolation.Quadratic),
                 new MagnitudeScoringFunction(
-                    fieldName: IndexFields.Search.TotalDownloadCount,
+                    fieldName: IndexFields.Search.LogBase2DownloadCount,
                     boost: 100.0,
-                    parameters: new MagnitudeScoringParameters(2000, double.MaxValue, shouldBoostBeyondRangeByConstant: true),
-                    interpolation: ScoringFunctionInterpolation.Logarithmic)
+                    parameters: new MagnitudeScoringParameters(
+                        boostingRangeStart: 0,
+                        boostingRangeEnd: Math.Log(999_999_999_999, 2),
+                        shouldBoostBeyondRangeByConstant: true),
+                    interpolation: ScoringFunctionInterpolation.Linear)
```

Comment:
The `LogBase2DownloadCount` field has a smaller range than `TotalDownloadCount`, which makes comparing its boosted score to the `Published` freshness score significantly easier.